grafana-plugin/src/pages/users/Users.tsx

```diff
@@ -323,6 +323,10 @@ class Users extends React.Component<UsersProps, UsersState> {
 
   renderNote = (user: UserType) => {
     const { store } = this.props;
+
+    const telegramConfigured = store.teamStore.currentTeam?.env_status.telegram_configured;
```

Comment:
telegram is always enabled in cloud envs, but we don't need to show the warning if the organization haven't connected the bot